docs/install.md

```diff
@@ -3,26 +3,80 @@ id: install
 title: Install
 ---
 
-Install with `yarn`:
+First, install Prettier locally:
 
-```bash
-yarn add prettier --dev --exact
-# or globally
-yarn global add prettier
+```sh
+npm install --save-dev --save-exact prettier
```

Comment:
Did you consider using [this feature](https://docusaurus.io/docs/en/doc-markdown#language-specific-code-tabs) of Docusaurus for Yarn commands.

Reply:
Didn’t know about that feature! Should we use it?

Reply:
We could try.

Reply:
Added.